service/worker/batcher/activities.go

```diff
@@ -386,13 +529,14 @@ func (a *activities) getOperationConcurrency(concurrency int) int {
 func startTaskProcessor(
```

Comment:
No need to change this IMHO, it's never going to be called again and we should delete it once we know nobody is calling the old workflow directly anymore.